ADA/ada.qmd

```diff
@@ -19,57 +17,234 @@ It brings together:
+
+To access your data and make use of cloud computing resource, you will need access to a cluster, which is a group of virtual machines that complete your processing for you. Your cluster setup will depend on which programming language you wish to use. You will be given access to a shared cluster when your data access is granted. Please see the [how clusters work](https://analytical-data-access.education.gov.uk/get-support/how-clusters-work) page on the ADA guidance site for more detail. 
+
+---
+
+#### Writing and running code: notebooks
+
+---
+
+In the Databricks interface, you have the choice to use notebooks, scripts or the Databricks SQL Editor. 
+
+Notebooks are easy to use in Databricks, and might be something you've not come across before. Notebooks are made up of multiple code or markdown cells, which may mean that you have to structure your code in a different way than you're used to. They are different to IDEs (Integrated Development Environments) like RStudio. You can find more information on Databricks notebooks and their structure on the [ADA guidance site](https://analytical-data-access.education.gov.uk/get-support/using-r-python-and-sql-in-databricks).
+
+The table below, taken from [NHS-R guidance](https://nhsdigital.github.io/rap-community-of-practice/implementing_RAP/notebooks_versus_ide_development/), shows a comparison of the features of notebooks and IDEs to help you decide where each would be most appropriately used:
+
+| Feature | Notebooks | IDEs |
+| --------|--------|---------|
+| Interactive outputs (tables, plots, etc) | Yes | No |
+| Storytelling and sharing results | Yes | No |
+| Benefits out-of-the-box (minimal configuration) | Yes | No |
+| Deterministic outputs (i.e. same result every run) | No | Yes |
+| Supports modular code structure & packages | No | Yes |
+| Supports unit testing frameworks | No | Yes |
+| Nice version control (e.g. readable git diffs) | No | Yes |
+| Autocomplete / auto-formatting & syntax highlighting | Yes* | Yes |
+| Compatible with sharing code externally | Yes** | Yes |
+  
+*= auto-formatting is available for SQL and Python in Databricks, but not R  
+**= external users must have software that would allow them to open notebooks
+
+Notebooks can be helpful for ad hoc, exploratory, or one off pieces of analysis to add additional documentation as you go, but please consider the skill sets and knowledge levels in your team when doing so. You can also use notebooks to present findings to others by hiding code cells and only showing Markdown cells containing charts, tables, images or text. They can be exported as HTML files to enable easier sharing outside of the Databricks interface. 
```

Comment:
I think the tone of the 'but please' is slightly off with the way we're trying to pitch this - maybe drop that half of the sentence and instead move up the sentence lower down in this section that advises caution / thinking of who will be picking up the work at a later point?

Reply:
When I moved that bit up it didn't sit right in that paragraph anymore so I've separated it and put it in a callout box. Thoughts?

Reply:
Like it - callout box works ideally for it